frontend/src/components/PathAppLayout.tsx

```diff
@@ -0,0 +1,110 @@
+  path: 1 | 2 | 3 | 4;
+};
+
+export const PathAppLayout: React.FC<PathAppLayoutProps> = ({ path }: PathAppLayoutProps) => {
+  const [pageNum, setPageNum] = useState<0 | 1 | 2 | 3 | 4 | 5>(0);
+
+  const next = () => {
+    if (pageNum < 5) {
+      setPageNum((prevPageNum) => (prevPageNum + 1) as 0 | 1 | 2 | 3 | 4 | 5);
+    }
+  };
+
+  const back = () => {
+    if (pageNum > 0) {
+      setPageNum((prevPageNum) => (prevPageNum - 1) as 0 | 1 | 2 | 3 | 4 | 5);
+    }
+  };
+
+  const path_descriptions = {
+    1: (
+      <p>
+        Path 1 is for Applicants who meet minimum education or experience-only requirements but who
+        are still in the process of completing the necessary education/work experience, may take the{" "}
+        <span className={styles.red}>IDEX California® Examination</span>. After successfully
+        passing the <span className={styles.red}>IDEX California® Examination</span>, these
+        candidates must submit the required proof of education/work experience, as listed under the
+        Categories of Certification, in order to complete the certification process.
+      </p>
+    ),
+
+    2: (
+      <p>
+        Path 2 is for Applicants who meet minimum education and education/work experience, may take
+        the <span className={styles.red}>IDEX California® Examination</span>. After successfully
+        passing the <span className={styles.red}>IDEX California® Examination</span>, these
+        candidates must submit the required proof of education/work experience, as listed under the
+        Categories of Certification, in order to complete the certification process.
+      </p>
+    ),
+
+    3: (
+      <p>
+        Path 3 is for Applicants who meet minimum education and education/work experience, and has
+        passed one of the qualifying National Interior Designer Exams. Applicants are eligible and
+        must take the <span className={styles.red}>IDEX California® Examination</span>. After
+        successfully passing the <span className={styles.red}>IDEX California® Examination</span>,
+        these candidates must submit the required proof of education/work experience, as listed
+        under the Categories of Certification, in order to complete the certification process.
+      </p>
+    ),
+
+    4: (
+      <p>
+        Path 4 is for Applicants who meet minimum education and education/work experience, have
+        passed one of the qualifying National Interior Designer Exams, and passed required ICC
+        courses. Applicants are eligible and must take the{" "}
+        <span className={styles.red}>IDEX California® Examination</span>. After successfully
+        passing the <span className={styles.red}>IDEX California® Examination</span>, these
+        candidates must submit the required proof of education/work experience, as listed under the
+        Categories of Certification, in order to complete the certification process.
+      </p>
+    ),
+  };
+
+  return (
+    <Page>
```

Comment:
I think we can not use the `<Page>` component for this page. Unfortunately its applying a width rule to the children that we don't want for this specific page. If you want to still sue the page component you can see how `additionalStyles` is implemented in the `<Button>` component to see how you can edit the styling of `<Page>`  component